spejstore/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: "3"
+services:
+  db:
+    image: ghcr.io/hackerspacekrk/spejstore/postgres-hstore:latest
+    restart: always
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    volumes:
+      - spejstore-pg:/var/lib/postgresql/data
+  web:
+    image: ghcr.io/hackerspacekrk/spejstore:latest
+    restart: always
+    command: bash -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
+    secrets:
+      - spejstore_clientid
+      - spejstore_secret
+    environment:
+      SPEJSTORE_CLIENT_ID: $$(cat /run/secrets/spejstore_clientid)
+      SPEJSTORE_SECRET: $$(cat /run/secrets/spejstore_secret)
+    ports:
+      - "8021:8000"
+    depends_on:
+      - db
+    env_file:
+      - spejstore.env
+    labels:
+      - "traefik.enable=true"
+      - "traefik.docker.network=system_default"
+      - "traefik.http.services.spejstore.loadbalancer.server.port=1880"
```

Comment:
Which port number is used? 8000/8021 or 1880?

Reply:
Good one, 8021 (as SSO was on 8037)